### ISSUE TITLE
How to import excel sheets in collins ??

### ISSUE THREAD

**Issue by rajatm0303** [tumblr/collins#274]
I want to use collins to keep information of machines in my lab. I used to use excel sheets to maintain details for every system but now I want to use collins to do so. All my systems have already assigned IP configuration parameters.

Thanks

**Comment by Primer42** [comment]
Hi @rajatm0303 
Probably your best option is to write a small script that reads your exel file, and then makes the correct collins API calls to create the assets, and populate their attributes.
We have a pretty well supported client library for Ruby, and there's one or two go clients that I've seen, though the API isn't all that complicated.
[Here is some documentation for the API.](http://tumblr.github.io/collins/api.html)
Please let me know if you run into any issues!

**Comment by asheepapart** [comment]
CC: @nadeemahmad
Hi @rajatm0303,

We have a small PHP library that we wrote to parse excel files and import them into Collins using the API. It works pretty smoothly for us. We get the spreadsheet from our vendor and then upload it to a PHP web app, which parses it and creates the New assets in Collins. 

We open sourced an Excel parsing library (see https://github.com/box/spout), but the glue code we wrote to tie into Collins is still not ready for show time. LMK if you're interested in early access.

From,
--- Ben
